Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -547,7 +547,7 @@ void BluetoothRealDevice::SaveLinkKeys()
     oss << Common::MacAddressToString(address);
     oss << '=';
     oss << std::hex;
-    for (const u16& data : entry.second)
+    for (u8 data : entry.second)
```

Comment:
Note that this will print out as a character, rather than as a value, like it would with u16. Maybe cast data to u16 in body if that matters.

Reply:
I went with the cast, with a comment to explain why.